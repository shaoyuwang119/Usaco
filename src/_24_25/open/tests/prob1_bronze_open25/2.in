90 95
D
DD
LDD
DDWD
DDWWD
DWDWDD
WLWLWLD
LDWLLDDD
WWWDLWDWD
DWDDDDLDWD
DWLLWWWDDLD
DLDLLWLDWDDD
WDWWLLDLLDWLD
LWDLWWDWWLDWWD
LLLWLLDLLLWDLWD
WDWLWWDWWLDDWLDD
DLLDWLWWLDDWDWWDD
WDDLLLWLDLLLDLDDWD
WWLLDDLDWWLWLWWDLWD
LLDDDDWDDLDDWLWWDLWD
LLLDDDDDDLLLLDDDWWLLD
DLWLLWDLLLLWLDDDLWWLDD
LWDDLWDWDLDWLWWDWDWDWDD
DDDLDDDWWLWWLLLLWWLLDDDD
LLWDLDWLLDDDDLDDLWDDLDLWD
DWLLWLDWWDLWLWDWDDDWLWDDDD
WWWWDDLLWLDWLLDDWLWLDWLDLDD
DLDDLDWDDDDLWWWWDLDLDWWDLLLD
WDWDWWLDWLLDWWLWDWWDWWLLLDLWD
DDWLWDWLWLDDWLLLLWWDDDDWDWLWWD
LDDLDLWWDDDWDDLWDLDLWWWLDDWLDLD
WDDWLDLLDWWWWLLDDLDWWDLDDWWWDDDD
DLLDLWLWLLDDLWDLDDDWLDDDWWLLLWWDD
WDLWDLWWLLDLWWLDLDLDWWLDDDDLWLDWDD
DWDLWWDLLLWDLDWDDLDLWLDWWLDLWLLWWLD
WDWWLDWLDLLDLLDDDWWDWWWDDWWLLLLWWLLD
WLLDDLLWDWLWWLDWWWLLDDDWWLLWLDLLLWWWD
DLDWLLDLDWLDDLLDDLDWDLLDLLLDWLWDLLLLLD
DLDWWLWWLWWDLLWDLWWWWLDDWWWDDWWDLDLWDLD
DLWDDDDWWLDLLDLDWWLDDWLDDLWWWLLDDDDDWWWD
WLLLWDDWLLLLLDLDDLDLWLDWWWLDWLLDLWWLLWWDD
DDWWWLWLLWDDDWWLWWLLWDLDWLLWLWWWDWDLLWLDLD
DWLWWWLWDDLLDLLLLLLDWLDDDLDWDWDDWLLWDDDWWWD
DLDDWLDWDLWWLWWWWWWLDWLDLLLDLDWWWWLWWDWDDLDD
LLWWWWLWLLDWDWWWWDDDDLLWLDWDWWDLLWWWDLLDWDDLD
DLWWLLLLLWDWWWDDDLLWLLDDDLLWDLWWLLLLDLWDWWLLDD
LLWDWLWWDLDLDLWLDWLWDLLLWLDWLDWLDWLDWDLLLDDWLDD
DWDWWLLWDDLWLLLLWWWLWDDLLDWDWDDLLDLLLDLWLDWLWDWD
WWWLLWWWLDLDWLWWWLLLLWDDWLLLWWWDDWDWWDDWLDLLWWDDD
DDDWLLDLWLLWLWDWDLLDDLWLLLDWWLWDDWWDLDDDWWWDLDLLWD
WLLLWWDLDLWLDDLLLWWDDWWLDWWLLDWDLDDLWLLDLDLDLDLLDWD
DLDLWWLDLDLLWDWLLLLLLWWLWLDWLDWLLDDWLWWWLWWLDDLDLDDD
DLDLDDWWWLLLWWWLDLLLLWWDWWWDLLWLWDDWLWWWDWLLLDWWDWDDD
WDWLLWDWDLWWWWWLLDLWDDDWDWWWDDWLWWLLLWDWDWWLLLDLLWDLWD
DLWDLDDWWWDDDDDWWDDLWWWWLDLWDDDLDDWWWDDWLLDWWWLWLDDDDDD
WWDDDLDLDWLDWDLLLDLDWLWLLLWDLLDDWDLLLWLDDWLWWWWDLWWWLLLD
DWLWWWDDLWLDLDDWDWLDWDWDLDWLLDDLDWLDDWLLDDDWDLWLWLDDDLLDD
WWWDLWWWWWLDDDWDDLDWWDWDLDDLWWLWDDLDWWDLWWDWWDLLDWWDLDDLWD
WLWDLWDDDLLDWWLWDLLLWWDLWLDLLLWWDDLDDLLLDLDWLDLLDWDLDDWDLLD
WDDWWLLLWWLWDWDLWWWLLDWLDDWLLLLWLDWWDLLDLDDLLDWDLLWDWDDDWLWD
LDLLLWWDLWDDLDWLDLLWDLLLLDDLWDWDLLDWLDWWWWWLLDDDDLDWLLDDWLDDD
DDLDWDLLDDLLWLDDWDWLLDWWWLDWLWLDDWWLWLLDDLDWDWWLDWWWDLLWLWWDLD
WDLLLWDLLLDWDLDWWWDWDWWWDWLWWLLLLWWLWWDWLLWDDLLDWWDWWDWWWDWLWDD
WDLDDDLDLDDWWLWWWWWWWLDWDLWWLDLWDDDDDLLLDWWWWWWDWLLDLWDDLLWLLDLD
WLWLLDDDLWDDLDLWDDLDLLDDWLLWWLWLWLWWLDWLLDWDLDLLDLDWDLDDDWWWWDWLD
WWWLWDLLWDDWDWDWLWDDLDDLWLLLDLWWDLLLLWLLWDDWDWLWWLLWWWLWWDDLLWLWLD
LDLLDDWWLDLWWDDLDLWWDDLWWLWWLDDDWWDLWLDDWLDDDDWLWLDDDLLLWLLDLLDDWWD
DLWWLLDWWWDDLDWDWDWDDDDWWDLDWLLLLLWDLWDDWDLWWDLWWWLWWDWDLWDWLDWLDDWD
WWLDLDWDDWWWLWLLLWLLWWLWDWWDWWDLWLLDDWLWWLLLDLWDWWDLDDLDLWWLWLLLWWLWD
WWLLWWLLDDLWLDDLWDWDWDDWWLWDDWLDWDLLDLLWDDWDLDLLDLLLWLDLWLDLDLLWDDWDLD
WLDLWLWLDDLLDLLDWLDDLLDDDLLLLWDLDWWWWDWWDWWDWWDDLWDWDWLWLWLDLWLLLDDWWWD
LLLLLLLDLLDDLLLWWDLWDLLLLLLDLDLDLDDLDWDLWDWWWLLWLDLLLLDWDLWDLLDDDDWWDLLD
LLDWDLDWWLLDLLDDLWLDLLLWWDDDDDLLWDLLLWWDWLDWWWDLDLWWWWLWWDDDLWDDWWLDWDLDD
DDDDWDWLWDWDLWWDLLWDWDLDWDDDWLLWDWDWLDLLLDWDDLDLDLLWLDLWDWDLLDLLDLDDWWDDWD
WLDLWWLDDLDDWWLWWWWLLLWDLLDLLWDLLDDLWDLWWWWWLWLLWWDLLWWDLDLDWWWWWDLLLDWLWDD
WDLDLDWWLWDLDDLWLWLWDDDDDLWDDWLDDWLLLWLLLDDLLDDDDDWWDDLLLWLWWLDDLLDLDDLWDLWD
LWWDDWDLDDLLWDLWDLWLLDWWWWLDDDLDLDDDDWLLLDWDDDDDWLWWWDWLLDLLWWLDWLDLDLWLDWLLD
LDWLDWDDWDLLLLWLWWLLWLWDWWWLLDWLWWDDWLWDDWLDWWDDLWWDDLWWDLDLLLWWWDWLLDWWLDLWLD
DLWDDDWDWLWLLDLWLLWDDLWLLWLWLWLWDWWWDDWWWWWLLLWLLLWLDDDLLWLWWDWDLWWWDLWDWWWWWLD
DWDWWDDDLWWLWDDDLWLDDLDWLLWWLDDDDDWDWWWDLWWLDLDLWWDWDLDDDDDWDLWLWDWLDDLLWLLDWDWD
DWLDLWLWWLWLWDWDLLDLLDDLLDWDDDLLLLLLWDLLWLWLLDLLLWWWLDDDWLLDWLDDDDDLDLDWLLLLDLDDD
LLWDLLWLLLDWWLDWLLWDLDWLWDDDWDLWLWWDDLLLWLWDLLWWLWDWDWWWLWLLWLWLWDWWWDLDLDWDWDWLDD
WLWLDLLWWDDWWLDDWLDWLDDLLLWLLDLLLWDDDWWWDLDDDWLLLLWWLLWWLLDWLWLWLWDDDWDDDWWWDWWDLLD
WDLDLDLWLWWWLDWLDWDDDLLLWWLDLWLLLLLWWWDLWDLWLLLDLWDDLWLLWDDDLWLDLLDLWWDWWWDLLDLDDDWD
DDDWWWDDDDLDLLLWDDLWWLWDWLLDLDLDDDLDWLDLDLDWDDDDDWDDLLDWWLDDDLDWWDLWLWWLWLLDLDDDWLWDD
DWDWWLWLLWWWLLDDLLDDWDLWLLLLWDDWLLWDWLLWWWWLDLLWLDWWWDLWLDLLWWLDLDLWLWWDDDLDLWWDWWDDWD
DLLWDLWLWWDLWWWDDLWLLDWWDDLWDLDDLLDLWWLLWWWWWLDLLLDDLLWDLWDWDDWLDWWDDLDWLWDWDLDLWWDWWLD
DLDDDDDLWDDLWWLWWLLDWWWDWDWWLDDLDWWWWLDWLWDLWLDWDWLLWLWWDWDWWWLLLDWDWLLDDLWLDLDLLWWLDLWD
WWLLDWDLWLLDLWDLDLLLWWDLDWDWLLDDLWWDDLDLWWWLLDLLWDDLDDDLWLLDDWWDLDWWLLDLWLWDDWWWLDLWLWDDD
DLDDDDLWWLDWLWLWLLWDLLDDLLDDDWWLWDWWLWDDWLDDLLLLDLDWLLLWWDDLLDDLDDLDDLWLWWDLWLDWLDLWLLDLLD
28 48
44 59
17 70
46 68
43 38
45 16
20 15
88 28
36 52
54 52
64 25
74 12
46 64
1 30
90 34
62 67
9 62
22 39
15 26
80 59
60 62
66 36
69 11
62 33
2 54
39 10
38 34
1 40
63 46
13 13
52 82
35 67
33 8
17 10
59 33
87 70
58 33
2 62
59 72
32 61
67 40
56 7
8 34
88 15
63 29
38 51
81 72
44 17
80 32
21 45
85 61
81 61
25 5
15 78
31 65
6 78
38 71
73 40
9 63
64 35
35 36
41 77
80 64
76 2
8 29
15 30
46 68
10 60
19 28
74 86
51 2
88 27
88 65
63 90
73 90
74 47
56 36
53 54
14 5
16 26
8 58
61 3
61 17
31 42
81 45
78 72
49 16
57 46
27 74
17 11
35 12
66 77
54 47
90 68
10 53
