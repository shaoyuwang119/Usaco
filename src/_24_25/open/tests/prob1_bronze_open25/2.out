1044
2171
1376
1700
2475
1211
1539
2016
1700
2016
1376
2324
1539
1700
2475
2171
1700
1376
1376
1211
1859
2016
1539
1211
1044
875
2016
1700
1539
4736
2171
2171
1700
875
1376
1376
1211
1539
2171
1539
1539
2171
2171
1859
1211
2016
3059
1859
704
2171
1859
2324
2171
1700
1211
2324
2324
2016
1859
1211
2624
2624
875
1211
1700
1539
1700
1539
1539
1859
1376
1859
1539
1376
2624
1539
2475
1700
1700
704
1376
2475
1859
1700
2324
2016
2171
1044
2171
531
2016
1539
1539
1859
1044
